test/test.js

```diff
@@ -68,6 +71,12 @@ describe('chai-immutable (' + typeEnv + ')', function () {
       it('should fail using `not` given an empty collection', function () {
         fail(function () { expect(new List()).to.not.be.empty; });
       });
+
+      if (typeEnv === 'Node.js') {
```

Comment:
I couldn't figure out how to load a different copy of Immutable for the Phantom tests, but I don't think it should really be a problem anyway since in a browser everything will just use whatever's on the `window` object. I also don't love wrapping `it`s inside a conditional — let me know if there's a way you'd rather I write this!